components/tools/OmeroWeb/test/integration/test_api_containers.py

```diff
@@ -156,6 +171,22 @@ def screen_plates(self, user1):
 
         screen = get_update_service(user1).saveAndReturnObject(screen)
         plate = get_update_service(user1).saveAndReturnObject(plate)
+
+        # Add well to first plate
```

Comment:
I will update ```test_api_wells.py``` to test with & without PlateAcquisitions, but it doesn't make sense for every test to test everything. Otherwise we will never stop writing tests and all the tests will be so large that you can't understand them.